### ISSUE TITLE
Loss won't decrease. 

### ISSUE THREAD

**Issue by Thiago-Meyer** [mozilla/DeepSpeech#2114]
For support and discussions, please use our [Discourse forums](https://discourse.mozilla.org/c/deep-speech).

If you've found a bug, or have a feature request, then please create an issue with the following information:

- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository) - No **:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04) CentOS**:
- **TensorFlow installed from (our builds, or upstream TensorFlow) pip (installed tensorflow gpu **:
- **TensorFlow version (use command below) 1.13.1**:
- **Python version 3.6**: 
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version 10**:
- **GPU model and memory - 2x Tesla V100 12G each**:
- **Exact command to reproduce**:

You can obtain the TensorFlow version with

```bash
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
```

I've started the training using the common voice data, following the example in the master branch. Each epoch took approximately 40 minutes to complete training and validation. But the loss won't decrease, I've trained 40 epochs, and the loss keeps around 190. 

I have used params as default, only changing the drop_rate to 0.30. I really don't know what to do to decrease the loss. After the training, the only thing that was recognized is " e e e e e e e e e e e e ".

**Comment by kdavis-mozilla** [comment]
Could you post your training log please?

**Comment by Thiago-Meyer** [comment]
I can't, sorry... Because I was using the screen visualization from my server. 
But after searching some things around the discussion area, I noticed some people were using the alphabet parameter. So I decided to use it, and guess what? My loss reduced to 50 and it's now converging.
